wiki/Sources.md

```diff
@@ -0,0 +1,213 @@
+# Source Geometry 
+
+A number of packages need to deal with fault geometry including:
```

Comment:
Should this be "A number of packages **needed**..." ?

Reply:
Yes it should be.

Reply:
actually no it not should be.